src/widgets/hits-per-page/hits-per-page.tsx

```diff
@@ -101,11 +101,12 @@ const hitsPerPage: HitsPerPageWidget = function hitsPerPage(
     cssClasses,
   });
 
-  const makeHitsPerPage = connectHitsPerPage(specializedRenderer, () =>
-    render(null, containerNode)
+  const makeHitsPerPage = connectHitsPerPage<HitsPerPageWidgetOptions>(
+    specializedRenderer,
+    () => render(null, containerNode)
   );
 
-  return makeHitsPerPage({ items, transformItems });
+  return makeHitsPerPage({ items, transformItems } as HitsPerPageWidgetOptions);
```

Comment:
The `renderFn` function is mistyped, we shouldn't have to cast the connector params. Ideally we should be able to do:

```js
  return makeHitsPerPage<HitsPerPageWidgetOptions>({ items, transformItems });
```

`HitsPerPageWidgetOptions` can then be used to type `widgetParams` in the connector and renderer.